src/main/scala/io/mesosphere/mesosaurus/Mesosaurus.scala

```diff
@@ -120,16 +121,16 @@ object Mesosaurus extends Logging {
         addOption(parser, MEM).`type`(UnsignedLong).help("mean # MB of memory")
         addOption(parser, MEM_SIGMA).`type`(UnsignedLong).help("memory standard deviation in MB")
         addOption(parser, PORT).`type`(UnsignedInteger).help("framework port to use")
-    }
+        addOption(parser, FAIL).`type`(UnsignedDouble).help("failure rate to use")
```

Comment:
It was meant to be just a decimal representing amount of tasks to fail i.e. 0.1 is 10%. Will give it a more specific name.